charts/internal/gvisor/templates/psp-gvisor.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: gvisor
```

Comment:
Multiple gvisor installations for different worker pools will all reconciles/overwrite this object, right?

Reply:
Yes. All workerPool dependent resources are moved to chart "gvisor-installation". 
This enables workerPool independent installation and deinstallation.
If all resources would be in the same chart/managed resource that would mean, when you remove one worker pool to use gvisor, it would remove the RuntimeClass etc. We also only want to have one RuntimeClass for gvisor.

Reply:
I know, but it's kind of odd. Then we could have also kept the solution as it was before. Mainly the same concerns expressed below in the actuator files... :|